packages/grafana-ui/src/components/InfoBox/InfoBox.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { GrafanaTheme } from '@grafana/data';
+import { css, cx } from 'emotion';
+import { stylesFactory, useTheme } from '../../themes';
+
+export interface InfoBoxProps {
+  header?: string | JSX.Element;
+  footer?: string | JSX.Element;
+  className?: string;
```

Comment:
use React.forwardRef instead (with Div element),then no need to add className here

Reply:
Hm, I'm not sure how `forwardRef` can help here if I need to pass `className` and merge it with existing styles? I'm looking into `Button` sources and there's className there:
https://github.com/grafana/grafana/blob/49a90cd764f1a57d6e03205747547d93525e27b0/packages/grafana-ui/src/components/Button/Button.tsx#L133-L145

Reply:
className is already defined on the element you are forward reffing: 

example here: https://github.com/grafana/grafana/blob/master/packages/grafana-ui/src/components/IconButton/IconButton.tsx#L11

Reply:
Ah, right, got it!